FFXIVClientStructs/FFXIV/Client/UI/UIClipboard.cs

```diff
@@ -0,0 +1,30 @@
+using FFXIVClientStructs.FFXIV.Client.System.Input;
+
+namespace FFXIVClientStructs.FFXIV.Client.UI;
+
+// Client::UI::UIClipboard
+// ctor "E8 ?? ?? ?? ?? EB 03 48 8B C6 4C 8B 87"
+[StructLayout(LayoutKind.Explicit, Size = 0xF8)]
+public unsafe partial struct UIClipboard {
+    [FieldOffset(0x8)] public UIModule* UIModule;
+    [FieldOffset(0x10)] public ClipBoard Data;
+    [FieldOffset(0xE8)] public ulong Hwnd;
+    [FieldOffset(0xF0)] public ulong HwndNext;
```

Comment:
Same here; hwnd values should be nint types.

```c
typedef PVOID void*;
typedef PVOID HANDLE;
typedef HANDLE HWND;
```
From [Windows Data Types](https://learn.microsoft.com/en-us/windows/win32/winprog/windows-data-types).